lib/modules/manager/gitlabci/common.ts

```diff
@@ -37,5 +37,8 @@ export function isGitlabIncludeLocal(
 export function isGitlabIncludeComponent(
   include: GitlabInclude,
 ): include is GitlabIncludeComponent {
-  return !is.undefined((include as GitlabIncludeComponent).component);
+  return (
+    !is.undefined((include as GitlabIncludeComponent).component) &&
+    typeof (include as any).component === 'string'
+  );
```

Comment:
Is `is.string(.....)` an alternative here?

Reply:
no any cast.